docs/getting-started.md

```diff
@@ -4,39 +4,43 @@ category: 5d4c940cae4e610060475769
 ---
 
 <p align="center">
-  <img width="350" src="./images/logo.svg" />
+  <img width="400" src="./images/logo.svg" />
 </p>
 
 <p align="center">
-  Automatic SDK generation from an OpenAPI definition.
+  Magical SDK generation from an OpenAPI definition 🪄
```

Comment:
wanna update the description in the package file too?

Reply:
done: https://github.com/readmeio/api/pull/487/commits/52bdd7791374ab400fc1fbf5a2ac59104f868fcf